src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java

```diff
@@ -239,14 +237,14 @@ private InstructorCourseDetailsPage getCourseDetailsPage() {
         return loginAdminToPage(detailsPageUrl, InstructorCourseDetailsPage.class);
     }
 
-    private boolean didStudentReceiveReminder(String courseName, String courseId, String studentEmail,
-                                              String studentPassword)
+    // TODO replace with GAE LocalMailService stub as we are checking the logic here, not reliability of email delivery
```

Comment:
Oh I see. Is it right to test email delivery though? Because one example of a point of failure is where a user receives more than 5 emails in 5 seconds. While it does not happen commonly,  the possibility makes the test fail when it should not.